main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -1300,19 +1300,15 @@ object Weeder2 {
       }
 
       mapN(
-        annVal,
+        annVal, // Even though the annotation is unused, we still need to map over it to collect possible errors.
```

Comment:
My plan was to allow people to write them and handle them in Weeder but have no effect but maybe that's a bad idea?

Reply:
You want to give a specific error message or?

Reply:
How about:

- Drop the comment
- Add an error saying that annotations are not allowed here

Reply:
I'm confused... the code stays more or less the same if we want to give an error saying that is not allowed. This comment is there because we do not use the `annVal` for anything other than including it in the map call so we collect its errors if there are any.

Reply:
It will go away when we introduce a `SharedContext`

Reply:
Ok. Then leave it